MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -2717,23 +2771,133 @@ Go Comprehensive API
 The Go Comprehensive API is a project for the future, to follow the C
 `Comprehensive API`_
 
-Go Utility Functions and Methods
-================================
+Go Utility Functions
+====================
 
------------------------
-Go KeyT Utility Methods
------------------------
+------------
```

Comment:
I thought we were getting rid of all the YDB prefixes on these routines? This was because the routine name is actually for example yottadb.YDBExit().

Reply:
We did. You were commenting on a line that was there in the previous version and removed in the commit!